src/api.ts

```diff
@@ -42,23 +57,33 @@ export const getImageName = (hash: CommitHash) => `${TAG_PREFIX}:${hash}`;
 export const extractCommitFromImage = (imageName: string): CommitHash => imageName.split(':')[1];
 
 /**
- * Returns which images are stored locally.
- * Polls locker docker daemon for image list
+ * Polls the local Docker daemon to
+ * fetch an updated list of images
  */
-export const { refreshLocalImages, getLocalImages } = (function() {
-	let localImages = {};
-	return {
-		refreshLocalImages: async () => {
-			localImages = _.keyBy(await docker.listImages(), 'RepoTags');
-		},
-		getLocalImages: () => localImages,
-	};
-})();
+export async function refreshLocalImages() {
+	const images = await docker.listImages();
+	const nextImages = new Map();
```

Comment:
can avoid mutation if you use the `Map` constructor
```js
const nextImages = new Map(images.map(image => [ image.RepoTags, image ] ) 
```

Reply:
good call

Reply:
refactored in 23525f4